rllib/agents/ars/ars_tf_policy.py

```diff
@@ -11,7 +11,7 @@
 from ray.rllib.policy.sample_batch import SampleBatch
 from ray.rllib.utils.filter import get_filter
 from ray.rllib.utils.framework import try_import_tf
-from ray.rllib.utils.space_utils import unbatch
+from ray.rllib.utils.spaces.space_utils import unbatch
```

Comment:
Can we call this tensor_utils or something? I'd like to avoid overloading the term "space" which is only used by gym.

Reply:
Concretely, I would place this file in `rllib/utils/tensor_utils.py`